BookingsAPI/Bookings.DAL/Queries/GetParticipantWithSuitabilityAnswersQuery.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Bookings.DAL.Queries.Core;
+using Bookings.Domain.Participants;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Bookings.DAL.Queries
+{
+    public class GetParticipantWithSuitabilityAnswersQuery : IQuery
+    {
+        private int _limit;
+
+        public GetParticipantWithSuitabilityAnswersQuery()
+        {
+            Limit = 100;
+        }
+
+        public string Cursor { get; set; }
+
+        public int Limit
+        {
+            get => _limit;
+            set
+            {
+                if (value <= 0) throw new ArgumentException("Limit must be one or more", nameof(value));
+                _limit = value;
+            }
+        }
+    }
+
+    public class GetParticipantWithSuitabilityAnswersQueryHandler : IQueryHandler<GetParticipantWithSuitabilityAnswersQuery,
+            CursorPagedResult<Participant, string>>
+    {
+        private readonly BookingsDbContext _context;
+
+        public GetParticipantWithSuitabilityAnswersQueryHandler(BookingsDbContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<CursorPagedResult<Participant, string>> Handle(GetParticipantWithSuitabilityAnswersQuery query)
+        {
+            IQueryable<Participant> participants = _context.Participants
+                .Include("Person")
+                .Include("HearingRole")
+                .Include("Hearing.HearingCases.Case")
+                .Include("SuitabilityAnswers");
+
+            participants = participants.Where(x => x.SuitabilityAnswers.Any()).OrderByDescending(x => x.SuitabilityAnswerUpdatedAt).ThenBy(x => x.Id.ToString());
+
+            if (!string.IsNullOrEmpty(query.Cursor))
+            {
+                TryParseCursor(query.Cursor, out var updatedDateTime, out var id);
```

Comment:
Partly for the things @rehmanab mentions below but also for less duplication, higher reuse, putting the validation earlier, readability etc, you could break the cursor out into a separate class which you convert the string cursor from in the controller. That way it's clear that this takes a cursor and if you pass in an invalid cursor you get a proper `BadRequest` exception